src/Tools/Source/OpenSourceDebug/OpenSourceDebug.csproj

```diff
@@ -31,14 +31,13 @@
     <VSSDKTargetPlatformRegRootSuffix>Roslyn</VSSDKTargetPlatformRegRootSuffix>
     <SolutionDir Condition="'$(SolutionDir)' == '' OR '$(SolutionDir)' == '*Undefined*'">..\..\..\..\</SolutionDir>
     <NonShipping>true</NonShipping>
+    <DeployExtension Condition="'$(DeployExtension)' == ''">True</DeployExtension>
```

Comment:
It's a "fun" MSBuild feature. Any /p set on command line force-overrides all writes to the property unless you explicitly go out of your way to [say otherwise with TreatAsLocalProperty](https://msdn.microsoft.com/en-us/library/ms171458.aspx#Anchor_4).